src/libpriv/rpmostree-scripts.c

```diff
@@ -506,9 +507,11 @@ impl_run_rpm_script (const KnownRpmScriptKind *rpmscript,
   if (headerGet (hdr, rpmscript->progtag, &td, (HEADERGET_ALLOC|HEADERGET_ARGV)))
     args = td.data;
 
+  const rpmFlags flags = headerGetNumber (hdr, rpmscript->flagtag);
```

Comment:
Nit/question: it looks like there is no NULL-check on `rpmscript` before dereferencing to its field, neither here nor a couple of lines earlier in this function, but I don't know if this kind of check is delegated to a single chokepoint earlier.

Reply:
See 

```
  const char *script = headerGetString (hdr, tagval);
  if (!script)
    return TRUE;

```

right?

Reply:
I think that's a different thing, but I walked a couple of callers up and this is implicitly covered by https://github.com/projectatomic/rpm-ostree/blob/master/src/libpriv/rpmostree-scripts.c#L779. Sorry for the noise.